examples/vite/src/App.vue

```diff
@@ -0,0 +1,87 @@
+<script setup lang="ts">
+import { Contract } from 'crossbell.js'
+import { ref, onErrorCaptured } from 'vue'
+import { useLocalStorage, useEventListener, useDark } from '@vueuse/core'
+import { type Address } from 'abitype'
+
+useDark()
+useEventListener(window, 'error', (event) => showResult(event))
+useEventListener(window, 'unhandledrejection', (event) =>
+  showResult(event.reason.toString()),
+)
+onErrorCaptured((err) => {
+  showResult(err)
+})
+
+const metamask = window.ethereum
+
+const contract = new Contract(metamask)
+const address = useLocalStorage<Address>('address', '0x')
+const characterId = useLocalStorage('characterId', '')
+const handle = useLocalStorage('handle', '')
+const result = ref('')
+
+async function connect() {
+  await showResult(contract.walletClient!.requestAddresses())
```

Comment:
Can we remove `!` for user?

Reply:
Maybe no, related to generic `BaseContract`